src/main/java/frc/robot/SubsystemManager.java

```diff
@@ -45,7 +45,7 @@ public void ZeroSensors() {
   /*
       Run Subsystem check command on all subsystems
   */
-  public void CheckSubsystems() {
+  public void checkSubsystems() {
```

Comment:
If you're making changes to this section of the code, may as well make the subsystem method `checkSubsystem()`

Reply:
Fixed (in other files)